pull_request_template.md

```diff
@@ -1,6 +1,23 @@
 # Purpose of this PR
 
+
+## Type of change
+
+- [ ] Bug fix 
+- [ ] Refactoring
+- [ ] New feature 
+- [ ] Minor change
+- [ ] Fundamental change
+- [ ] This change requires a documentation update
+
+
 ## Checklist:
-* updated in code documentation
-* adjusted reporting if needed
-* checked that REMIND still compiles
+
+- [ ] My code follows the coding ettiquette
+- [ ] I have performed a self-review of my own code
+- [ ] Changes are commented, particularly in hard-to-understand areas
+- [ ] I have updated the in code documentation
```

Comment:
I would write "in-code documentation" like [github does](https://coderefinery.github.io/documentation/in-code-documentation/)